docs/trace/usage.rst

```diff
@@ -209,6 +209,40 @@ requests will be automatically traced.
     # `GoogleCloudFormatPropagator` as propagator.
     middleware = FlaskMiddleware(app)
 
+The middleware also supports the factory pattern.
+.. code:: python
```

Comment:
Add a blank line before the code block? This part is not parsing by github correctly.